src/test/test-cgroup-util.c

```diff
@@ -392,18 +393,18 @@ TEST(cg_get_keyed_attribute) {
 
         assert_se(cg_get_keyed_attribute("cpu", "/init.scope", "cpu.stat", STRV_MAKE("usage_usec", "no_such_attr"), vals3) == -ENXIO);
         assert_se(cg_get_keyed_attribute_graceful("cpu", "/init.scope", "cpu.stat", STRV_MAKE("usage_usec", "no_such_attr"), vals3) == 1);
-        assert_se(vals3[0] && !vals3[1]);
-        free(vals3[0]);
+        assert(vals3[0] && !vals3[1]);
```

Comment:
Why do you convert `assert_se()` into `assert()`? `assert()` may be disabled when -DNDEBUG is provided to the compiler command line option. Moreover this is a testing code, please use `assert_se()`.